etcdserver/apply.go

```diff
@@ -72,9 +73,17 @@ func (s *EtcdServer) applyV3Request(r *pb.InternalRaftRequest) *applyResult {
 	ar := &applyResult{}
 	switch {
 	case r.Range != nil:
-		ar.resp, ar.err = s.applyV3.Range(noTxn, r.Range)
+		if s.AuthStore().IsRangePermitted(r.UserID, string(r.Range.Key)) {
```

Comment:
Yes, but current permission model doesn't support range well (as we discussed here: https://github.com/coreos/etcd/pull/4966#issuecomment-206741618). I'd like to add the range support in the near future, is this ok?